packages/fish.rb

```diff
@@ -46,6 +46,17 @@ def self.install
   end
 
   def self.postinstall
+    puts 'Installing Oh My Fish...'.lightgrey
+    system 'curl https://raw.githubusercontent.com/oh-my-fish/oh-my-fish/master/bin/install | fish'
+    puts 'Setting up profile...'.lightgrey
+    system 'omf install foreign-env'
+    FileUtils.mkdir_p '~/.config/fish/config.fish'
+    @fishconfig = <<~FISHCONFIGEOF
+      # Source variables written to ~/.profile
+      fenv source ~/.profile
+    FISHCONFIGEOF
+    File.write('~/.config/fish/config.fish', @fishconfig)
```

Comment:
Should we be writing to `#{CREW_DEST_HOME}` instead of `~` so this config is included in the package files?  Actually, shouldn't we be writing to `#{CREW_DEST_PREFIX}/etc/env.d` instead?

Reply:
I'm guessing we can bypass the entire `fenv` thing simply by writing the output of that command to the appropriate file?

Reply:
Not exactly-- that'll only fix whatever what was in ` ~/.profile` at the time. Any new packages that write to `~/.profile` (or bashrc, which several packages do) will have their changes ignored if the user is using fish.

Reply:
None of our packages should not be writing to `~/.profile` or `~/.bashrc` but rather to `#{CREW_DEST_PREFIX}/etc/env.d` instead.

Reply:
I count around 25 doing so and another 10 suggesting you do so. I think that's out of the scope of this PR, though.

Reply:
Don't look at other packages as an excuse to not fix this package.  I agree other packages need updated and we'll get to those as updates are available.  FYI, a lot of those packages were added prior to the `/usr/local/etc/env.d` addition.